apps/mark-scan/backend/src/app.ts

```diff
@@ -80,6 +81,7 @@ export function buildApi(
   usbDrive: UsbDrive,
   logger: Logger,
   workspace: Workspace,
+  chromium: Browser,
```

Comment:
Slight preference for `browser` or `renderer` rather than `chromium`.

Reply:
Yeah I was going back and forth on this one - will switch!